enhancements/installer/ovirt-rhv-upi.md

```diff
@@ -0,0 +1,166 @@
+---
+title: ovirt-rhv-upi
+authors:
+  - "@gekorob"
+  - "@rolfedh"
+reviewers:
+  - "@sdodson"
+  - "@abhinavdahiya"
+approvers:
+  - TBD
+creation-date: 2020-05-06
+last-updated: 2020-05-15
+status: implementable
+---
+
+# oVirt-RHV UPI
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Ovirt support for OpenShift 4 was initially focused on the installer provisioned
+(IPI) workflow.
+
+This enhancement proposes adding tooling and documentation to help users deploy
+OpenShift 4 on existing _user-provisioned infrastructure_ (UPI) in oVirt.
+
+## Motivation
+
+Most users who want to deploy OpenShift on oVirt/RHV have a highly-customized
+infrastructure. We can help these users simplify the process of installing OpenShift
+on oVirt using existing user-provisioned infrastructure such as DNS, DHCP, Load Balancers.
+
+Currently, users who want to deploy OpenShift on existing infrastructure in oVirt must
+adapt the [bare metal UPI instructions][baremetal-upi], which are not optimal for oVirt.
+
+Using the current installation program for deploying OpenShift 4 on oVirt does not work
+because it creates an opinionated set of _installer-provisioned infrastructure_ (IPI).
+Though convenient, this IPI does not meet the needs of users who want to use _existing_
+infrastructure.
```

Comment:
Lets make sure that we detail which portions of the IPI installation we feel is not compatible with customer use cases. UPI initially came about in a time where the Installer had zero support for pre-existing infrastructure but now most platforms support installation into existing VPCs and Networks (AWS terms, sorry) which allows IPI to be used in many more situations. We need to be tracking this so that we can make informed decisions as to which additional IPI customization we should explore.

Reply:
I agree that this is key point to discuss, thanks for point this out.